models/users.js

```diff
@@ -40,6 +40,7 @@ const addOrUpdate = async (userData, userId = null) => {
 
     // Add user
     userData.incompleteUserDetails = true
+    userData.createdOn = Date.now()
```

Comment:
@anuragKhandelwal1 , rather than this, can you check if there is any way in FireStore to automatedly add `createdAt` and `updatedAt` timestamps implicitly?

Reply:
Would `firebase.firestore.Timestamp` help? 
(Someone in discord channel said you shouldn't be including firebase just for this.)

Reply:
cc: @swarajpure

Reply:
We're now using `firebase-admin` package. Can you please check if we can use the implicit method now?